src/test/java/com/snack/news/config/MetaControllerTest.java

```diff
@@ -34,19 +35,22 @@
 
 	private MockMvc mockMvc;
 
-	@Before
-	public void setUp() {
+	@BeforeEach
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/manual/snack-news/Snack-BE/pullRequest?prid=4764524)